### DISCUSSION TITLE
Can't get rid of notifications from a repo

### DISCUSSION THREAD

**Discussion by nickali** [community/community#38174] [Other Feature Feedback, Questions, & Ideas]
I decided to unwatch all the repos that I've been monitoring over the years. I was able to clear the list of repos and got rid of the notifications.

Except for [one](https://github.com/serhii-londar/open-source-mac-os-apps). Here is what I see when I go to the repo:

![Screen Shot 2022-11-03 at 10 32 36 AM](https://user-images.githubusercontent.com/1514992/199749373-55f664bb-c65b-4a0c-8113-661c63adef0e.png)

That looks ok. But when I go the notifications page, I still see it listed on the sidebar:

![Screen Shot 2022-11-03 at 10 28 19 AM](https://user-images.githubusercontent.com/1514992/199748013-05cec6d5-55f8-48a3-bf33-0c18cf8dfbf0.png)

And perhaps most weirdly, my notification count is wonky:

![Screen Shot 2022-11-03 at 10 29 05 AM](https://user-images.githubusercontent.com/1514992/199748479-6241f21b-6004-4045-9608-40831acfa566.png)

Obviously, I don't get how I can have 1-0 of 1 notifications. 

Is there a way to fix this?

**Comment by david-engelmann** [comment]
[Do you see it listed when you go to this url?](https://github.com/notifications?query=is%3Aunread)

**Reply by nickali** [reply]
Yes, that's where I see the repo listed in the sidebar and the 1-0 of 1 notifications.

**Comment by DSDmark** [comment]
how strange, did you really not **done your notification**.

![Screenshot 2022-11-03 at 20-59-40 Notifications](https://user-images.githubusercontent.com/85744816/199764529-5e7283ef-ac30-4b56-ac8a-b1af60a421df.png)

in that case. I think it is some technical issue

**Comment by jfullstackdev** [comment]
not very sure of that but there is the ghost notifications that you can clear by this one 

https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

been using it for several times, it's always working

**Reply by nickali** [reply]
It worked to get rid of the blue dot by the notifications bell. Still, my count is of. It's progress.

**Comment by karlhorky** [comment]
What worked for me (which marks everything as read):

1. Go to Inbox
2. "Select all" (checkbox)
3. "Select all xxx notifications" (link)
4. Three dots menu (button)
5. "Mark as read" (menu item)

Although if the link in step 3 above doesn't show up for you, it's possible this works only with more than 25 notifications in Inbox (also including non-ghost notifications).
